### DISCUSSION TITLE
SSR takes a long time on Windows

### DISCUSSION THREAD

**Discussion by archiewood** [evidence-dev/evidence#2379] [General]
(Timebox)
## Steps To Reproduce

```
npx degit evidence-dev/template my-project
cd my-project
npm i
npm run dev
```

It takes ~20s to launch the dev server

### Environment

- Node version (`node -v`):
- npm version (`npm -v`):
- OS: Windows
- Browser:

## Expected Behavior

SSR is fast and page opens in ~5s 

## Actual Behaviour

SSR is slow and takes ~20s

**Comment by RandomFractals** [comment]
On Windows 10 with NodeJS v16.17:

![evidence-start-stop-server-status](https://github.com/evidence-dev/evidence/assets/656833/598edf04-87e1-44e1-a1c1-909f0ee51289)

**Comment by archiewood** [comment]
Perhaps tailwind the culprit:
https://github.com/vitejs/vite/issues/5145#issuecomment-930762504

**Comment by csjh** [comment]
Seems like the super long delay right after `npm i` is from dependency pre-bundling -- though I think vitejs/vite#12441 fixes/speeds up? When I override to 4.3.9 it's around 20s vs. 40 before. Still not sure about the shorter-but-still-long delay for subsequent runs

**Comment by hughess** [comment]
Here's some terminal feedback I got from a fresh install - this is on Mac, but I assume still applicable here.

This was without touching anything after starting the server. Looks like it ran the query 4 times upon server start:

<img width="1557" alt="CleanShot 2023-06-23 at 13 59 26@2x" src="https://github.com/evidence-dev/evidence/assets/12602440/823e08d2-5fcd-478c-86b1-5e0693ddc149">
